src/main/webapp/app/shared/metis/post/post.component.ts

```diff
@@ -107,6 +120,33 @@ export class PostComponent extends PostingDirective<Post> implements OnInit, OnC
         this.postFooterComponent.openCreateAnswerPostModal();
     }
 
+    /**
+     * sorts answerPosts by two criteria
+     * 1. criterion: resolvesPost -> true comes first
+     * 2. criterion: creationDate -> most recent comes at the end (chronologically from top to bottom)
+     */
+    sortAnswerPosts(): void {
+        if (!this.posting.answers) {
+            this.sortedAnswerPosts = [];
+            return;
+        }
+        this.sortedAnswerPosts = this.posting.answers.sort(
```

Comment:
Shouldn't we avoid in-place sorting?
 Maybe we can implement it like that:
` this.sortedAnswerPosts = this.posting.answers.slice().sort((a, b) => 
`